src/Admin/UserManager.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * Class UserManager.
+ *
+ * @since 1.6.0
+ *
+ * @package AMP
+ */
+
+namespace AmpProject\AmpWP\Admin;
+
+use AmpProject\AmpWP\Service;
+
+/**
+ * Class UserManager
+ *
+ * @since 1.6.0
+ */
+final class UserManager implements Service {
```

Comment:
Perhaps instead of `UserManager` we should consider something more explicitly like `DevToolsUserAccess`, so this is scoped specifically for that one concern.

Reply:
I like this direction.